interactions/api/http/message.py

```diff
@@ -54,6 +55,9 @@ async def send_message(
         if message_reference:
             payload["message_reference"] = message_reference
 
+        if stickers:
+            payload["sticker_ids"] = [sticker.id for sticker in stickers]
```

Comment:
doesn't it need to be int/str?

Reply:
It needs to be a string.

Reply:
Then change it please, also you got merge conflict

Reply:
Fixed the string part.

Tried fixing the merge conflict. Is it ok now?

Reply:
Looks like, yeah